src/app/clusters/ota-requestor/OTARequestor.cpp

```diff
@@ -270,6 +271,20 @@ void OTARequestor::ConnectToProvider(OnConnectedAction onConnectedAction)
                    ChipLogError(SoftwareUpdate, "Cannot establish connection to provider: %" CHIP_ERROR_FORMAT, err.Format()));
 }
 
+// Application directs the Requestor to cancel image update in progress. All the Requestor state is
```

Comment:
nit: Probably no need to repeat the same comment as what's already in the header?

Reply:
I actually have the opposite view on this: I find it inconvenient that people don't comment function definitions. When looking through the code and trying to understand what a function does one has to stop and chase the function declaration -- potentially through multiple header files and complicated chains of inheritance.

Reply:
I find it extra maintenance. If the comment changes in one spot, you need to remember to change it in the other place. Many times, only one has been done so you end up with out of sync comments. Nonetheless, this was a nit.

Reply:
Yes, producing readable code requires extra effort.